pycape/cape.py

```diff
@@ -387,6 +387,9 @@ async def _request_connection(self, function_ref, pcrs=None):
                 )
         return
 
+    async def _close(self):
+        await self._ctx.close()
+
```

Comment:
this should be removed, not sure when it was added back in

Reply:
oops will be removed. thanks